### DISCUSSION TITLE
16 Pod per node limit?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [nestybox/sysbox#616] [General]
> With Sysbox, you can only run 16 Sysbox pods per Kubernetes worker node.

Will this limitation reconsidered? It's pretty much saying it's enterprise only product, no? It can't even run on GKE where it has 15+ daemonsets by default. 🙄

**Comment by ctalledo** [comment]
Hi @shinebayar-g, yes this limitation will be removed before the next Sysbox release (Jan/Feb 2023).

**Comment by alasdairtran** [comment]
Hi @ctalledo, I was wondering if the 16-pod limit is still in the latest 0.6.1 release?

**Comment by rodnymolina** [comment]
@alasdairtran, the 16-pod limitation has been eliminated in the latest v0.6.1 release. Thanks for the reminder. I have just updated our `Changelog` file to reflect this important point.
